app/views/checkouts/confirm.html.haml

```diff
@@ -0,0 +1,107 @@
+= content_for(:title, t('order.checkout') + ' | ')
+%main.container.general-main-wrap
+  %h1.general-title-margin= t('order.checkout')
+  %ul.steps.list-inline
+    %li.step.done
+      %span.step-number
+        %i.fa.fa-check.step-icon
+      %span.step-text.hidden-xs= t('checkout.address')
+    %li.step-divider
+    %li.step.done
+      %span.step-number
+        %i.fa.fa-check.step-icon
+      %span.step-text.hidden-xs= t('checkout.delivery.delivery')
+    %li.step-divider
+    %li.step.done
+      %span.step-number
+        %i.fa.fa-check.step-icon
+      %span.step-text.hidden-xs= t('checkout.payment.payment')
+    %li.step-divider
+    %li.step.active
+      %span.step-number 4
+      %span.step-text.hidden-xs= t('checkout.confirm')
+    %li.step-divider
+    %li.step
+      %span.step-number 5
+      %span.step-text.hidden-xs= t('checkout.complete')
+
+  .row.mb-20
+    .col-sm-3
+      %h3.general-subtitle= t('checkout.address')
+      %p.general-address
+        = "#{current_user.billing.first_name} #{current_user.billing.last_name}"
+        %br
+        = current_user.billing.address
+        %br
+        = current_user.billing.city
+        %br
+        = current_user.billing.country
+        %br
+        = current_user.billing.phone
+
+    .col-sm-3
+      %h3.general-subtitle= t('checkout.delivery.delivery')
+      %p.general-address
+        = current_order.delivery.method
+        %br
+        = current_order.delivery.days
+
+    .col-sm-3
+      %h3.general-subtitle= t('checkout.payment.payment')
+      %p.general-address
+        = current_user.credit_card.number
+        %br
+        = current_user.credit_card.expiration_date
+
+  .hidden-xs.divider-lg.pt-20
+    %table.table.table-hover
+      %thead
+        %tr
+          %th
+            %span.in-grey-600= t('order.product')
+          %th.col-method
+          %th
+            %span.in-grey-600= t('order.price')
+          %th
+            %span.in-grey-600= t('order.quantity')
+          %th
+            %span.in-grey-600= t('order.subtotal')
+      %tbody
+        - current_order.order_items.each do |order_item|
+          %tr
+            %td
+              .general-img-wrap-table
+                - if order_item.book.images.any?
```

Comment:
It looks like, you use this logic it several places. I suppose that is time to create helper or partial for this.